### ISSUE TITLE
Can you tell me the accuracy of hmdb51?

### ISSUE THREAD

**Issue by seominseok0429** [LossNAN/I3D-Tensorflow#23]
Can you tell me the hyperparameter?
